kaldb/pom.xml

```diff
@@ -575,6 +575,9 @@
                     <version>2.22.2</version>
                     <configuration>
                         <trimStackTrace>false</trimStackTrace>
+                        <systemPropertyVariables>
```

Comment:
damn! when i was trying I did the same but called the property `log4j2.configurationFile` based on theri docs :(